docs/fundamentals/diagnostics/sample/src/DiagDemo/Startup.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Diagnostics;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Http;
+using Microsoft.Framework.DependencyInjection;
+using System;
+
+namespace DiagDemo
+{
+    public class Startup
+    {
+        // For more information on how to configure your application, visit http://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
+        {
+        }
+
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        {
+            if (string.Equals(env.EnvironmentName, "Development", StringComparison.OrdinalIgnoreCase))
+            {
+                app.UseErrorPage(ErrorPageOptions.ShowAll);
+
+                app.UseRuntimeInfoPage(); // default path is /runtimeinfo
+            }
+            else
+            {
+                // specify production behavior for error handling, for example:
+                // app.UseErrorHandler("/Home/Error");
```

Comment:
Why is this commented out? You could send it to the welcome page for demo purposes.